pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -634,7 +639,7 @@ contract StablePhantomPool is IRateProvider, BaseGeneralPool, ProtocolFeeCache {
         bytes memory userData
     ) private returns (uint256, uint256[] memory) {
         (uint256[] memory amountsIn, uint256 minBPTAmountOut) = userData.exactTokensInForBptOut();
-        InputHelpers.ensureInputLengthMatch(_getTotalTokens() - 1, amountsIn.length);
+        InputHelpers.ensureInputLengthMatch(balances.length - 1, amountsIn.length);
```

Comment:
Why the removal of `_getTotalTokens()` throughout this PR?

Reply:
I'd thought we were trying to deprecate calling that: but maybe that was just for managed pools? I don't feel strongly. Can put them back if you prefer. (It does save a little bytecode, but we're not in trouble there yet.)

Reply:
Hm I don't recall trying to remove that. I'd use `_getTotalTokens()` - it'll be cheaper and make intent more obvious.